app/src/components/notifications/LNotification.vue

```diff
@@ -41,8 +83,11 @@ const show = ref(true);
                 <div class="ml-4 flex flex-shrink-0">
                     <button
                         type="button"
-                        @click="show = false"
+                        @click="
+                            notification.id ? removeNotification(notification.id) : (show = false)
```

Comment:
A notification always has an id, so this will always remove the notification. The reason why this only toggles `show` was that the `NotificationManager` is responsible for deleting the actual notification from the store after a while anyway, which I think could be fine to keep, what do you think?